lib/player.rb

```diff
@@ -0,0 +1,60 @@
+require 'awesome_print'
+require_relative '../lib/scoring.rb'
+require 'pry'
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :total
+    attr_accessor :tiles
+    def initialize(name)
+      @name = name
+      @plays = Array.new
+      @total = 0
+      @tilebag = Scrabble::Tilebag.new
+      @tiles = @tilebag.draw_tiles(7)
+    end
+
+    def play(word)
+      if word =~ [/[a-zA-Z]+/]
+        return nil
+      end
+
+      @plays << word
+      if won?
+        return false
+      else
+        return Scrabble::Scoring.score(word)
+      end
+    end
+
+    def total_score
+      @total = 0
+
+      @plays.each do |word|
+        @total += Scrabble::Scoring.score(word)
+      end
+
+      return @total
+    end
+
+    def won?
+      @total >= 100 ? (return true) : (return false)
```

Comment:
This code assumes that `total_score` has been called **before** `won?` has been called.  Not a good assumption!

Instead just call `total_score` in this method.